libraries/rush-lib/src/api/RushProjectConfiguration.ts

```diff
@@ -63,17 +63,6 @@ export interface IRushPhaseSharding {
    * @defaultValue `--shard-output-folder=.rush/operations/{phaseName}/shards/{shardIndex}`.
    */
   outputFolderArgumentFormat?: string;
-
-  /**
-   * Configuration for the shard operation. All other configuration applies to the collator operation.
-   */
-  shardOperationSettings?: {
```

Comment:
May be a good idea to keep this around with type `unknown` and emit a warning with instructions on how to move to the new format.

Reply:
Makes sense, updated in [a438175](https://github.com/microsoft/rushstack/pull/4923/commits/a438175472486a2e95b54cf695b9c1a0d6c4fe7f) to just be a deprecate notice and support both options